target/linux/ramips/dts/mt7628an_comfast_cf-wr617ac.dts

```diff
@@ -0,0 +1,86 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+/ {
+  compatible = "comfast,cf-wr617ac", "mediatek,mt7628an-soc";
+  model = "Comfast CF-WR617AC";
+
+  chosen {
+    bootargs = "console=ttyS0,115200";
+  };
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x4000000>;
+	};
+};
+
+&spi0 {
+	status = "okay";
+
+	m25p80@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <10000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot";
+				reg = <0x0 0x30000>;
+				read-only;
+			};
+
+			ubootenv: partition@30000 {
+				label = "u-boot-env";
+				reg = <0x30000 0x10000>;
+				read-only;
+			};
+
+			factory: partition@40000 {
+				label = "factory";
+				reg = <0x40000 0x10000>;
+				read-only;
+			};
+
+			partition@50000 {
+				compatible = "denx,uimage";
+				label = "firmware";
+				reg = <0x50000 0x760000>;
+			};
+		};
+	};
+};
+
+
+&pcie {
+  status = "okay";
+};
+
+&pcie0 {
+  wifi@0,0 {
+    compatible = "mediatek,mt76";
+    reg = <0x0000 0 0 0 0>;
+    mediatek,mtd-eeprom = <&factory 0x8000>;
+    ieee80211-freq-limit = <5000000 6000000>;
+  };
+};
+
+&ethernet {
+  mtd-mac-address = <&factory 0x4>;
+  mtd-mac-address-increment = <(-1)>;
```

Comment:
Please check whether there are addresses in 0x28, 0x2e, 0xe000 and 0xe006.

FYI: https://openwrt.org/docs/guide-developer/mac.address#find_out_about_flash_locations